### PR TITLE
Updated db connections with schema

### PR DESCRIPTION
**Pull request by sambuds**
Signed-off-by: Sam Sinha <ssinha484@gmail.com>

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -37,18 +37,18 @@ const monax = require('@monax/burrow');
...
-        );
-        _.set(
-          settings,
-          'monax.pg.chain_db_url',
-          `postgres://${process.env.POSTGRES_DB_USER}:${process.env.POSTGRES_DB_PASSWORD}@${process.env.POSTGRES_DB_HOST}:${process.env.POSTGRES_DB_PORT}/${process.env.POSTGRES_DB_DATABASE_VENT}`,
-        );
-      }
+      _.set(
+        settings,
+        'monax.db.app_db_url',
```
Comment (compleatang) [inline api/app.js]: think it should be `db.app_db_url` for consistency. similarly below

File: api/scripts/migrateUsers.js [hunk 2]
```diff
@@ -27,22 +27,18 @@ global.__monax_constants = require(path.join(__common, 'monax-constants'));
...
-                    process.env.POSTGRES_DB_HOST +
-                    ':' +
-                    process.env.POSTGRES_DB_PORT +
-                    '/' +
-                    process.env.POSTGRES_DB_DATABASE
-      )
-    }
+    _.set(
+      settings,
+      'monax.db.app_db_url',
```
Comment (compleatang) [inline api/scripts/migrateUsers.js]: think it should be `db.app_db_url` for consistency. similarly below

File: api/scripts/migrateUsers.js [hunk 3]
```diff
@@ -27,22 +27,18 @@ global.__monax_constants = require(path.join(__common, 'monax-constants'));
...
+      `postgres://${process.env.POSTGRES_DB_USER}:${process.env.POSTGRES_DB_PASSWORD}@${process.env.POSTGRES_DB_HOST}:${process.env.POSTGRES_DB_PORT}/${process.env.POSTGRES_DB_DATABASE}`,
+    );
+    _.set(settings, 'monax.db.app_db_schema', process.env.POSTGRES_DB_SCHEMA);
+    _.set(
+      settings,
+      'monax.pg.chain_db_url',
```
Comment (compleatang) [inline api/scripts/migrateUsers.js]: `db.chain_db_url`

File: api/scripts/migrateUsers.js [hunk 4]
```diff
@@ -27,22 +27,18 @@ global.__monax_constants = require(path.join(__common, 'monax-constants'));
...
+      `postgres://${process.env.POSTGRES_DB_USER}:${process.env.POSTGRES_DB_PASSWORD}@${process.env.POSTGRES_DB_HOST}:${process.env.POSTGRES_DB_PORT}/${process.env.POSTGRES_DB_DATABASE}`,
+    );
+    _.set(settings, 'monax.db.chain_db_schema', process.env.POSTGRES_DB_SCHEMA_VENT);
```
Comment (compleatang) [inline api/scripts/migrateUsers.js]: rm `monax.`

File: api/app.js (hunk 1 above)
Comment (sambuds) [reply api/app.js]: done

**Pull request merged**